qa/sc_cert_fee_conf.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python2
+import time
+from decimal import Decimal
+
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from test_framework.util import assert_false, assert_true, start_nodes, \
+    websocket_port_by_mc_node_index
+from SidechainTestFramework.scutil import bootstrap_sidechain_nodes, \
+    start_sc_nodes, connect_sc_nodes, generate_next_blocks, generate_next_block
+from SidechainTestFramework.sc_forging_util import *
+
+"""
+Check the bootstrap feature.
```

Comment:
First few lines of test description are outdated (header and configuration). Check them and change.